ofbizDemo/groovyScripts/FindProductionRun.groovy

```diff
@@ -11,24 +11,23 @@ condition = exprBuilder.AND() {
         EQUALS(productId: parameters.productId)
     }
     if (parameters.currentStatusId) {
-        IN(currentStatusId: parameters.currentStatusId)
+        if (currentStatusId == 1) {
+            IN(currentStatusId: parameters.currentStatusId)
+        }
+        EQUALS(currentStatusId: parameters.currentStatusId)
     }
     if (parameters.workEffortName) {
-        EQUALS(workEffortName: parameters.workEffortName)
+        EQUALS(workEffortName: parameters.workEffortName) LIKE(workEffortName: "%" + workEffortName + "%")
```

Comment:
This is also not correct condition
It should be like LIKE(workEffortName: "%" + workEffortName + "%") not EQUALS